src/MaaCore/Task/Roguelike/RoguelikeSettlementTaskPlugin.cpp

```diff
@@ -90,7 +90,7 @@ bool asst::RoguelikeSettlementTaskPlugin::get_settlement_info(json::value& info,
         const auto& task_replace = Task.get<OcrTaskInfo>(task_name)->replace_map;
 
         auto merge_map = std::vector(number_replace.begin(), number_replace.end());
-        merge_map.insert(merge_map.end(), task_replace.begin(), task_replace.end());
+        ranges::copy(task_replace.begin(), task_replace.end(), merge_map.end());
```

Comment:
不对吧，如果你用 ranges::copy 的话应该要 back_inserter 吧
```suggestion
        ranges::copy(task_replace, std::back_inserter(merge_map));
```